tests/compat.sh

```diff
@@ -74,8 +74,8 @@ PEERS="OpenSSL$PEER_GNUTLS mbedTLS"
 print_usage() {
     echo "Usage: $0"
     printf "  -h|--help\tPrint this help.\n"
-    printf "  -f|--filter\tOnly matching ciphersuites are tested (Default: '$FILTER')\n"
-    printf "  -e|--exclude\tMatching ciphersuites are excluded (Default: '$EXCLUDE')\n"
+    printf "  -f|--filter\tOnly matching ciphersuites are executed (BRE; default: '$FILTER')\n"
```

Comment:
Sorry, but what does BRE mean?

Reply:
It stands for Basic Regular Expression - as you know even in Posix there are two kinds of regular expressions: Basic and Extended, often referred to as BRE and ERE in Posix documents. I was trying to be precise (as "regex" could be any variety), but perhaps I shouldn't have used the abbreviation.

Reply:
Oh, thanks for the info, I wasn't aware of the acronym.